apps/cyberstorm-remix/app/root.tsx

```diff
@@ -142,13 +143,28 @@ export async function clientLoader() {
     publicEnvVariables.VITE_COOKIE_DOMAIN
   );
 
-  // We need to run this here too in addition to the, shouldRevalidate function,
-  // as for some reason the commtis to localStorage are not done before the the clientLoader is run
-  sessionTools.sessionValid(
-    publicEnvVariables.VITE_API_URL,
-    publicEnvVariables.VITE_COOKIE_DOMAIN
+  let forceUpdateCurrentUser = false;
+  if (
+    request.url.startsWith(`${publicEnvVariables.VITE_BETA_SITE_URL}/teams`) ||
```

Comment:
Why do we force this for these two pages? Or rather, why don't we force it on all pages? Why show any page to whoever is browsing as if they're a specific user, if we can't be sure they actually are that user? Or the other way around, why not let user know immediate when we detect that they're no longer authenticated?